modules/root/root_api.rb

```diff
@@ -19,7 +18,9 @@ class Proxy::RootApi < Sinatra::Base
 
   get "/version" do
     begin
-      {:version => Proxy::VERSION}.to_json
+      content_type :json
+      modules = Hash[::Proxy::Plugins.enabled_plugins.collect {|plugin| [plugin.plugin_name.to_s, plugin.version.to_s]}].reject { |key| key == 'foreman_proxy' }
+      {:version => Proxy::VERSION, :modules => modules}.to_json
```

Comment:
Q: does this means we are breaking the API ? what about foreman code where the proxies are not updated, should we start talking about API versions?